modules/distribution/src/main/scripts/fluo

```diff
@@ -13,73 +13,111 @@
    bin="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
    SOURCE="$(readlink "$SOURCE")"
-   [[ $SOURCE != /* ]] && SOURCE="$bin/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+   [[ $SOURCE != /* ]] && SOURCE="$bin/$SOURCE"
 done
-bin="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-script=$( basename "$SOURCE" )
-# Stop: Resolve Script Directory
-
-. "$bin"/impl/config.sh
+# Set up variables needed by fluo-env.sh
+export bin="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+export basedir="$( cd -P ${bin}/.. && pwd )"
+export conf="$basedir/conf"
+export lib="$basedir/lib"
+export cmd="$1"
+case "$cmd" in
+  oracle|worker) export app="$2" ;;
+esac
+export FLUO_VERSION=${project.version}
 
+if [ ! -f "$conf/fluo-env.sh" ]; then
+  echo "fluo-env.sh must exist in $conf"
+  exit 1
+fi
+source "$conf/fluo-env.sh"
 export CLASSPATH
 
+mkdir -p "$basedir/run" 2>/dev/null
+
+deprecated_fluo_props=$conf/fluo.properties
+
+if [[ -f "$FLUO_CONN_PROPS" && -f "$deprecated_fluo_props" ]]; then
+  echo "Fluo is being configured by $FLUO_CONN_PROPS and $deprecated_fluo_props. Remove one of these files."
+  exit 1
+fi
+
 # stop if any command fails
 set -e
 
-APP=$2
-APP_DIR=$FLUO_HOME/apps/$APP
-APP_CONF_DIR=$APP_DIR/conf
-APP_LIB_DIR=$APP_DIR/lib
-
 function copy_config {
-  if [ -f $FLUO_CONF_DIR/$1 ]; then
-    cp $FLUO_CONF_DIR/$1 $APP_CONF_DIR/
-  elif [ -f $FLUO_CONF_DIR/examples/$1 ]; then
-    cp FLUO_CONF_DIR/examples/$1 $APP_CONF_DIR
+  if [ -f "$conf/$1" ]; then
+    cp "$conf/$1" "$APP_CONF_DIR/"
+  elif [ -f "$conf/examples/$1" ]; then
+    cp "$conf/examples/$1" "$APP_CONF_DIR"
   else
-    echo "Config file $1 not found in $FLUO_CONF_DIR or $FLUO_CONF_DIR/examples"
+    echo "Config file $1 not found in $conf or $conf/examples"
     exit 1
   fi
 }
 
 function print_usage {
   echo -e "Usage: fluo <command> (<argument> ...)\n"
   echo -e "Possible commands:\n"
-  echo "  list          Lists all Fluo applications in Fluo instance"
-  echo "  new <app>     Creates configuration for new application in apps/"
-  echo "  init <app>    Initializes Fluo application using configuration in apps/<app>/conf/fluo.properties"
-  echo "  start <app>   Starts Fluo application on cluster"
-  echo "  stop <app>    Stops Fluo application on cluster"
-  echo "  kill <app>    Kills Fluo application on cluster"
-  echo "  status <app>  Prints status of Fluo application"
-  echo "  info <app>    Prints information about containers of Fluo application"
-  echo "  scan <app>    Prints snapshot of data in Fluo application"
-  echo "  classpath     Prints the classpath setup in fluo-env.sh"
-  echo "  wait <app>    Waits until all notifications are processed"
-  echo "  version       Prints the version of Fluo"
-  echo "  exec <app> <class>  {<argument>}";
+  echo "  init <appProps> {<arg>}     Initializes Fluo application using <appProps>. Run with -h to see additional args."
```

Comment:
I had a thought, not sure if it will work.  If this were of the form `init <app name> <appProps> {args}` then it would be more consistent with other commands and the fluo-app.props file would not require the name prop.